src/python/create.py

```diff
@@ -164,7 +168,13 @@ def create_materials(parsed_materials, project, data, citations):
             cell_value = parsed_cell["value"]
 
             if cell_type == "attribute":
-                material_dict[cell_key] = cell_value
+
+                if cell_key == "computational_forcefield":
```

Comment:
I think one line of comment above this might be helpful with something along the lines of

```python
# for the `input & output data` sheet, if it has a row of data, then that is related to the correct material and inputted into the material node as an attribute
```